core/src/main/java/io/javaoperatorsdk/jenvtest/process/ProcessReadinessChecker.java

```diff
@@ -179,6 +179,7 @@ public void checkServerTrusted(
           null);
       return HttpClient.newBuilder()
           .sslContext(sslContext)
```

Comment:
Would make also a comment here why we are doing this, with a reference to the feature flag

Reply:
Will do.